packages/comments/src/parts/comment-form.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Input, Spacer } from '@openstad-headless/ui/src';
+import { SecondaryButton } from '@openstad-headless/ui/src/button';
+import CommentFormPropsType from '../types/index';
+import DataStore from '@openstad-headless/data-store/src';
+import hasRole from '../../../lib/has-role';
+
+function CommentForm({
+  comment = {},
+  descriptionMinLength = 30,
+  descriptionMaxLength = 500,
+  placeholder = 'Type hier je reactie',
+  requiredUserRole = 'member',
+  ...props
+}: CommentFormPropsType) {
+
+  const args = {
+    comment,
+    descriptionMinLength,
+    descriptionMaxLength,
+    placeholder,
+    requiredUserRole,
+    ...props
+  } as CommentFormPropsType;
+
+  const datastore = new DataStore(args);
+  const [currentUser, currentUserError, currentUserIsLoading] = datastore.useCurrentUser({ ...args });
+
+  function canSubmit() {
+    return true;
```

Comment:
Can submit is altijd true, klopt dit?

Reply:
Voor nu: ja, omdat we nog geen errorHandling hebben. Deze functie zou clientside alvast moeten ondervangen wat er mis kan gaan, met name  descriptionMin/MaxLength, en dan een melding geven, maar daar moeten we dus nog een structuur voor opzetten.

Reply:
Nu geeft dit serverside errors (die overigens dus ook nog niet getoond worden).